### ISSUE TITLE
Mouse Lock going bananas(Atleast on MacOS)

### ISSUE THREAD

**Issue by BlackGoku36** [armory3d/armory#1370]
Thank you for contributing to Armory!

**Description**
Mouse lock on MacOS make it go crazy, it feels like it sensitivity is too high, move mouse a little and watch it do 360*. If you try FPS/TPS example, and lock mouse and try to move it, it will go bananas.

**To Reproduce**
Try FPS/TPS or any example with mouse locking and camera rotating.

**Expected behavior**
Moving mouse by little should do mouse moment little.

**System**
Blender: Blender 2.8
Armory: Latest git, updated 1-2 days ago
OS: macOS Mojave
Graphics card: Intel Iris Pro 1536 MB

**Test File**
Armory's FPS/TPS template

**Comment by onelsonic** [comment]
Hi there, I have the same issue on Mojave. 
I searched around and it seems there were some issues with Mouse controls and Privacy Accessibility with some games on Mojave.
Did it ever worked before for you with the previous Armory builts or not?
Were you on Mojave when it worked?

**Comment by BlackGoku36** [comment]
Yes, previous armory build worked even when I was on mojave. I am using catalina, and issue is still there

**Comment by Lykdraft** [comment]
I am on Mojave and got the same problem. Also tested with the FPS Template. It's *really* going bananas. You can say that. 
Looks like some kid swinging the mouse around like crazy. In the end it's always ending up with an upside down view of the scene when you let go of the button.
